packages/graphql-auth-transformer/src/resources.ts

```diff
@@ -900,7 +901,7 @@ identityClaim: "${rule.identityField || rule.identityClaim || DEFAULT_IDENTITY_F
       TypeName: subscriptionTypeName,
       RequestMappingTemplate: print(
         raw(`{
-    "version": "2018-05-29",
+    "version": "${RESOLVER_VERSION_ID}",
```

Comment:
One thought - Can we tell in general what versions customers are using in templates? For example if they manually create a resolver with the older version perhaps we could flag a warning similar to what we do today for types that are not annotated with `@auth`?

Also have we given any thought to implications for existing projects, or do we expect this to be transparent?

Reply:
iirc there isn't a mechanism in place which can run a LGTM for VTL. If anything we could the resolver changelog into the custom resolver docs.